app/frontend/src/theme.ts

```diff
@@ -26,6 +29,14 @@ declare module "@material-ui/core/styles/shape" {
 const themeOptions: ThemeOptions = {
   breakpoints,
   overrides: {
+    MuiCssBaseline: {
+      "@global": {
+        a: {
```

Comment:
Do we have any default link styles from design yet? It'd be good to include here too if they exist?

Reply:
I think it's better to put them in Mui <Link> or something - this is just a css reset

Reply:
Good point, yeah better to put in `MuiLink` instead